packages/events/src/tests/postgres-migrations.sql

```diff
@@ -134,6 +134,7 @@ CREATE TABLE app.event_actions (
     request_id text,
     status app.action_status NOT NULL,
     transaction_id text NOT NULL,
+    is_immediate_correction boolean,
```

Comment:
could/should we include this in the annotation instead, to avoid a new column in event_action table, which would be used only for this one type of action?